src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/DeleteSpecificCommentAuthorizationHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿#nullable enable
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
+{
+    public class DeleteSpecificCommentRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class DeleteSpecificCommentAuthorizationHandler
+        : AuthorizationHandler<DeleteSpecificCommentRequirement, Comment>
+    {
+        private readonly ContentDbContext _contentDbContext;
+        private readonly IReleasePublishingStatusRepository _releasePublishingStatusRepository;
+        private readonly IUserPublicationRoleRepository _userPublicationRoleRepository;
+        private readonly IUserReleaseRoleRepository _userReleaseRoleRepository;
+
+        public DeleteSpecificCommentAuthorizationHandler(ContentDbContext contentDbContext,
```

Comment:
Where are the tests?

Reply:
For some reason I assumed they were already written and I just needed to change the permission test - my bad